_includes/meta.html

```diff
@@ -50,4 +50,7 @@
     ================================================== -->
   {% asset index.scss %}
   {% asset video-js.min.css %}
+
+<!-- jQuery -->
+  <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.5.1/jquery.min.js"></script>
```

Comment:
@RSD-accessboard instead of adding it from an external CDN, just keep to the paradigm we've done with our other dependencies.  `npm install --save jquery` and then add it to the `_config.yml` like the `anchor-js`, and pull it in like `{% asset jquery.min.js %}` in the `_includes/scripts.html`